client/web/src/store.ts

```diff
@@ -5,16 +5,19 @@ import createSagaMiddleware from 'redux-saga'
 import { UserReducer, userSaga } from './features/user'
 import { seriesSaga } from './features/series'
 import { mirrorPacsSaga } from './features/mirrorPacs'
+import { labelsSaga } from './features/labels'
 import serieReducer from './features/series/seriesReducer'
 import { authSaga, AuthReducer } from './features/auth'
 import { VersionReducer, versionSaga } from './features/version'
+import labelsReducer from './features/labels/labelsReducer'
```

Comment:
you can move it to `import { labelsSaga } from './features/labels'`

Reply:
Done